elyra/pipeline/catalog_connector.py

```diff
@@ -83,17 +83,20 @@ class CatalogEntry(object):
     catalog_type: str
     runtime_type: RuntimeProcessorType
     categories: List[str]
+    file_extensions: str
 
     def __init__(self,
                  entry_data: EntryData,
                  entry_reference: Any,
                  catalog_instance: Metadata,
-                 hash_keys: List[str]):
+                 hash_keys: List[str],
+                 file_extensions: List[str]):
```

Comment:
The way it's currently set up, no because this is the object that we instantiate (not the one the catalog connector author instantiates). But this gave me an idea that I'm going to try